app/Main.hs

```diff
@@ -54,6 +55,8 @@ cliParser =
       ]
   seed :: Parser (Maybe Int)
   seed = optional $ option auto (long "seed" <> short 's' <> metavar "INT" <> help "Seed to use for world generation")
+  webPort :: Parser (Maybe Int)
+  webPort = optional $ option auto (long "web" <> metavar "INT" <> help "Start the web interface")
```

Comment:
If I saw the words "web interface" I would assume it is an entire alternative UI for playing the game.  Of course if people think that, they are going to be disappointed.  Maybe call it the `web debugging interface` to make it more clear what it is?